.github/agents/copilot-instructions.md

```diff
@@ -0,0 +1,31 @@
+# mac-test-reliability-fixes Development Guidelines
```

Comment:
I'm planning to remove all of the `.github`, `.specify`, and `.vscode` changes - so please ignore